oidc-api/src/test/java/uk/gov/di/authentication/oidc/services/QueryParamsAuthorizeValidatorTest.java

```diff
@@ -122,7 +122,6 @@ private static Stream<Arguments> invalidVtrAttributes() {
         return Stream.of(
                 Arguments.of(jsonArrayOf("Cm")),
                 Arguments.of(jsonArrayOf("Cl.Cm.P3")),
-                Arguments.of(jsonArrayOf("Cl.P0", "Cl.Cm.P2")),
```

Comment:
As far as I'm aware, this is a valid VTR. A VTR is a disjunction of Vectors of Trust, so we can choose one without an identity component VoTs with and without are present. I'm not sure if these tests cases are testing what was intended. I think perhaps the two below are checking if the order of the individual IDs matters. The Vector Of Trust spec says they shouldn't [(https://datatracker.ietf.org/doc/html/rfc8485).](url) The documentation on confluence says they should ([https://govukverify.atlassian.net/wiki/spaces/Architecture/pages/3322282099/Vectors+of+Trust](url) - Ps should always come before Cs and Cl should always come before Cm). In practise we don't care but these tests make it look like we do. The reason those VTRs are considered invalid is because P1 is set as unsupported and the client isn't set up to accept the HMRC LoCs.

Reply:
I think this is along the lines of removing the check that we don't want identity and non identity in the same request. It's a logical change to the code, and should be in a separate pr if we want to consider it

Reply:
Re-added with comments to clarify the reason for failures.